packages/components/src/CreditCard/Register.purs

```diff
@@ -0,0 +1,44 @@
+module KSF.CreditCard.Register where
+
+import KSF.User (PaymentTerminalUrl)
+import React.Basic as React
+import React.Basic (JSX, make)
+import React.Basic.DOM as DOM
+
+type Props = 
+  { title :: JSX
+  , terminalUrl :: PaymentTerminalUrl }
```

Comment:
Styling: Looks like our convention has been to put the closing bracket on multi line data definitions on its own line.